### ISSUE TITLE
Logistic models: Be less verbose by default + be less scary for users with less red

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/parameters#1034]
![image](https://github.com/user-attachments/assets/922e84dc-ef8f-4a17-be04-f0efdadf757e)

1) messages that are not warnings/errors and are *information* should be in yellow. Red really makes it look like something is wrong, and can frighten new users & students 👹 
2) I would remove the message about the exponentiate. While the one above is information, that one is a <sub>(debatable)</sub> suggestion and should be in docs

**Comment by bwiernik** [comment]
The red is just coming from your RStudio no? RStudio uses the same color for all messages, warnings, and errors.

**Comment by DominiqueMakowski** [comment]
I don't think these are messages per se, they are just part of the output, so we can specify colors as we often do elsewhere where we print info in yellow

**Comment by strengejacke** [comment]
Second is a message and only shown once per session: https://github.com/easystats/parameters/blob/034f5dfd763012bfca07c72724eb2f99c7e4dd86/R/format.R#L909

**Comment by strengejacke** [comment]
First one is also a message and not shown in red: 
https://github.com/easystats/parameters/blob/034f5dfd763012bfca07c72724eb2f99c7e4dd86/R/format.R#L857

**Comment by DominiqueMakowski** [comment]
I'd vote for:
1. Set the exponentiate warning to false by default (I don't think this info warrants an intrusive message, even once per sesh) 
2. Transform the information about how CIs/ps are computed as part of the pretty printing and display them in yellow

In general, we should probably take a look at homogenizing outputs / colour schemes, we have sometimes some green, yellow, reds, etc. arguably the problem with colours is that they must work on dark and white backgrounds, but I think if we stick to default for text, yellow for information, green/red for "passed/failed checks", and then warnings when actual warnings, we should be alright

**Comment by strengejacke** [comment]
What about model info? Yellow, too?

![image](https://github.com/user-attachments/assets/af5228fe-a5b8-4af3-9d5b-31567dc6338d)

**Comment by DominiqueMakowski** [comment]
it's a bit much, white is fine I'd say

Maybe we could add a "Note: " before "Uncertainty intervals..." just to suggest to participants that it's not an issue per se that it's just for their information

**Comment by strengejacke** [comment]
> Set the exponentiate warning to false by default (I don't think this info warrants an intrusive message, even once per sesh)

But then it's never shown.

You can always suppress all information with `verbose = FALSE`. And you can in general customize the amount of default information using global options:

https://easystats.github.io/parameters/reference/print.parameters_model.html#global-options-to-customize-messages-and-tables-when-printing

**Comment by DominiqueMakowski** [comment]
![image](https://github.com/user-attachments/assets/85355303-31f7-49dc-a4f1-02f1f23d716d)

**Comment by strengejacke** [comment]
Note that this note is useful for people less familiar with log(it) transformations! See also:

``` r
library(parameters)
model <- lm(log(mpg) ~ hp, data = mtcars)
model_parameters(model)
#> Parameter   | Coefficient |       SE |         95% CI | t(30) |      p
#> ----------------------------------------------------------------------
#> (Intercept) |        3.46 |     0.08 | [ 3.30,  3.62] | 44.04 | < .001
#> hp          |   -3.43e-03 | 4.87e-04 | [ 0.00,  0.00] | -7.05 | < .001
#> 
#> Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed
#>   using a Wald t-distribution approximation.
#> 
#> The model has a log-transformed response variable. Consider using
#>   `exponentiate = TRUE` to interpret coefficients as ratios.
```

``` r
library(parameters)
model <- lm(log(mpg) ~ hp, data = mtcars)
model_parameters(model, exponentiate = TRUE)
#> Parameter   | Coefficient |       SE |         95% CI | t(30) |      p
#> ----------------------------------------------------------------------
#> (Intercept) |       31.83 |     2.50 | [27.11, 37.37] | 44.04 | < .001
#> hp          |        1.00 | 4.85e-04 | [ 1.00,  1.00] | -7.05 | < .001
#> 
#> Uncertainty intervals (equal-tailed) and p-values (two-tailed) computed
#>   using a Wald t-distribution approximation.
#> 
#> The model has a log-transformed response variable, and exponentiated
#>   parameters are reported. A one-unit increase in the predictor is
#>   associated with multiplying the outcome by that predictor's coefficient.
```

![image](https://github.com/user-attachments/assets/6a0ae34d-488a-4544-9a12-9eb87278ae55)

**Comment by strengejacke** [comment]
And yes, your alerting red color is just because you have an RStudio setting that prints messages in red?!?

**Comment by DominiqueMakowski** [comment]
> Note that this note is useful for people less familiar with log(it) transformations! See also:

Yes but printing this in the output is IMO not the right place for this kind of information 🙃 

> you have an RStudio setting that prints messages in red?!?

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExeG13N3Fpejd3MHRoNzRyaXc2aGc0b3k0NWJsbjJqbHRlN2d2Y2N1cyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/DwIdasRkFKsMg/giphy.gif)

**Comment by strengejacke** [comment]
We could move parts of the information into the docs - what would be the best place? I'd say the `exponentiate` argumennt? https://easystats.github.io/parameters/reference/model_parameters.default.html

**Comment by bwiernik** [comment]
I would prefer to leave the exponentiate message, but print it as regular text (white or yellow) rather than as a `message()`

**Comment by strengejacke** [comment]
So we have mixed opinions on this issue. How do we decide?

**Comment by strengejacke** [comment]
Reference Screenshots: https://x.com/strengejacke/status/1850483090109432136

> I think the last one (large coefs) is most useful, and the one before (interpreting expt coefs) it least useful - IME it's unlikely to be clear enough to help someone who doesn't already get it, and not needed for someone who does

> I think none of the information is really worth printing each time. If it was all moved into the docs, that would be just as useful.

> Definitely keep that last sentence in the output for all cases you shared here.

> fwiw, I think that log(outcome) warning is gold. Here is a paper in the lancet insisting it's an odds ratio in the abstract, and I had to argue at length with a colleague that it was not in fact an OR ("But Darren, it's in The Lancet!")

> I prefer these informative messages and they're why I recommend the easystats packages when teaching. Students often struggle to identify the important information when it's buried in the docs. There's always summary() for "parsimonious" output. I'm not convinced how helpful the bottom left "interpretation" message is though (since it necessarily has to be fairly abstract), which makes me wonder the extent to which these messages actually improve model evaluation/interpretation. Would be interesting to test...

> I think it's nice to "warn" about exponentiating in both cases but I don't think the function should be giving interpretations. That should go in the docs, imo. I also think it's nice to warn about large coefficients.

>  I’m wary of interpretation stuff in general because it encourages not thinking for yourself, but at the very least hide it by default and have an argument like help=T to print it.

> As a non-statistician I find the messages on both left hand side images helpful.

**Comment by DominiqueMakowski** [comment]
the problem of exponentiate is that it might lead to misleading interpretations for models with interactions etc.

The last point is fair, and the idea of having a `help=TRUE` option to toggle actual advice could be interesting

**Comment by DominiqueMakowski** [comment]
@mattansb? also @profAndyField

**Comment by profandyfield** [comment]
The `help=T` suggestion was me, Dom. 'm generally with you in not wanting this sort of advice in the default output but instead think about ways to offer help to those that need it. Maybe eaststats need the MS paperclip (monster style, obvs)  .... 'It looks like you've fitted a logistic model ...' :-)

**Comment by mattansb** [comment]
I think all of these "tips" should appear as messages/warnings in the `model_parameters()` function (and can be disabled with `verbose=FALSE`, and not in the `print()`ed output/footer.
